web-components/src/components/combobox/ComboBox.ts

```diff
@@ -149,6 +150,16 @@ export namespace ComboBox {
     private multiSelected: number[] = [];
     private customContent: Element[] = [];
 
+    private notifylistitemscounttoparent() {
+      this.dispatchEvent(
+        new CustomEvent('item-count-changed', {
```

Comment:
rename item-count-changed to md-combobox-search-result-count

Reply:
Done